packages/genesys-spark-chart-components-react/package.json

```diff
@@ -13,7 +13,7 @@
     "clean": "rm -r ./dist || true",
     "eslint": "eslint . --ignore-path .gitignore --no-error-on-unmatched-pattern --fix",
     "lint-all": "npm-run-all \"eslint\" \"prettier\" \"prettier-package-json\"",
-    "lint-staged": "lint-staged",
+    "lint-staged": "lint-staged --concurrent false",
```

Comment:
There was a race condition as two commands were processing `package.json` at the same time.